### DISCUSSION TITLE
stuck in lesson10 of blockchain basics. my requests for eth sepolia keeps getting denied on Chainlink.

### DISCUSSION THREAD

**Discussion by sina2006-prog** [Cyfrin/foundry-full-course-cu#2838] [Q&A]
Hello every one. Well as I said my requests for eth sepolia keeps getting denied and then this error pops up that says: You must hold at least 1 LINK on Ethereum Mainnet to request native tokens. But as its been said in repo resources it seems it doesnt need you to hold anything in mainnet. I would really appreciate any helps. Thanks.

**Comment by EngrPips** [comment]
You can use this faucet https://sepolia-faucet.pk910.de/
